frontend/src/main/components/Courses/CoursesTable.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+
+
+
+export default function  CoursesTable({  courses, currentUser }) {
+
+   const limitText = ({value})=>{
+ 
+          
+            return value.substr(0,150);
+          
+          
+           
```

Comment:
I don't think all this whitespace is necessary?
```suggestion

export default function CoursesTable({ courses, currentUser }) {
    const limitText = ({value})=>{
        return value.substr(0,150);
```